k-distribution/include/kframework/builtin/kast.md

```diff
@@ -282,6 +282,17 @@ module RULE-CELLS
 
   syntax #OptionalDots ::= "..." [klabel(#dots), symbol]
                          | ""    [klabel(#noDots), symbol]
+
+  syntax Int
```

Comment:
Not happy about adding this sort here, but I understand the overall problem.
We might have issues if we want to restrict sorts to have a single point of origin.

Reply:
I mean, we're unlikely to do that because being able to declare sorts this way is kind of needed as a form of forward declaration to break certain types of cycles between modules.